Makefile

```diff
@@ -64,8 +64,9 @@ infra-set-up-account: ## Configure and create resources for current AWS profile
 	@:$(call check_defined, ACCOUNT_NAME, human readable name for account e.g. "prod" or the AWS account alias)
 	./bin/set-up-current-account.sh $(ACCOUNT_NAME)
 
-infra-configure-network: ## Configure default network
-	./bin/create-tfbackend.sh infra/networks default
+infra-configure-network: ## Configure network $NETWORK_NAME
+	@:$(call check_defined, NETWORK_NAME, the name of the network in /infra/networks)
```

Comment:
It looks like this defaults to matching the env in the app configs. Will that be clear to folks for how to set if they want something different? Or worth documenting? (For bid challenges, we would like something more like `navapbc-bb2` and for testing, things like `kmd-test` :) )

Reply:
network_name is only used for adding AWS tags. we'll also have project as a tag. i imagine we would set the project name to navabb2 or kmdtest. The network name doesn't actually need to be unique.